Indicators/SMA.py

```diff
@@ -1,5 +1,33 @@
 #Link:  https://www.investopedia.com/terms/s/sma.asp
+import pandas as pd
+
 
 # Simple Moving Average
-def SMA():
-    pass
+def SMA(data):
+
+    # Calculate SMA_Open
+    SMA_Open = [data['open'][0]]
```

Comment:
should be doing something like this 
`data['SMA_10'] = data.open.rolling(10, min_periods=1).mean()`
I would create a function that calls the SMA based on the period  as in 
`def SMA(column_data,period)`
and call like this
`aapl_data['new_col_name']=SMA(aapl_data['open'],20)`
which would return the new called data and be added to the df after excution

its important we do it this way because a lot of indicators use the SMA on different window intervals